src/assets/styles.scss

```diff
@@ -212,17 +230,26 @@ card-img-top {
     font-weight: 600;
 }
 
-/*
-   Button overrides;
-   more space between buttons on multiple consecutive lines
+/* 
+   Button overrides; 
+   more space between buttons on multiple consecutive lines 
    overall line-height is unchanged.
 */
 
 .btn {
```

Comment:
I would move the changes that are specific to the index page inside the container class. So we can create other buttons in the website and we can still modify their styles

Reply:
Okay, thank you!